web_src/js/features/dropzone.js

```diff
@@ -1,80 +1,131 @@
-import $ from 'jquery';
 import {svg} from '../svg.js';
 import {htmlEscape} from 'escape-goat';
 import {clippie} from 'clippie';
 import {showTemporaryTooltip} from '../modules/tippy.js';
-import {POST} from '../modules/fetch.js';
+import {GET, POST} from '../modules/fetch.js';
 import {showErrorToast} from '../modules/toast.js';
+import {createElementFromHTML, createElementFromObject} from '../utils/dom.js';
 
 const {csrfToken, i18n} = window.config;
 
-export async function createDropzone(el, opts) {
+async function createDropzone(el, opts) {
   const [{Dropzone}] = await Promise.all([
     import(/* webpackChunkName: "dropzone" */'dropzone'),
     import(/* webpackChunkName: "dropzone" */'dropzone/dist/dropzone.css'),
   ]);
   return new Dropzone(el, opts);
 }
 
-export function initGlobalDropzone() {
-  for (const el of document.querySelectorAll('.dropzone')) {
-    initDropzone(el);
-  }
+function addCopyLink(file) {
+  // Create a "Copy Link" element, to conveniently copy the image or file link as Markdown to the clipboard
+  // The "<a>" element has a hardcoded cursor: pointer because the default is overridden by .dropzone
+  const copyLinkEl = createElementFromHTML(`
+<div class="tw-text-center">
+  <a href="#" class="tw-cursor-pointer">${svg('octicon-copy', 14)} Copy link</a>
+</div>`);
```

Comment:
Would indent these, looks better and the few extra whitespace bytes don't matter.

Reply:
I think this layout is perfect and beautiful, and no extra space.

Reply:
I won't block but I still prefer optimizing for code readability.

Reply:
It does have the best readability


![image](https://github.com/go-gitea/gitea/assets/2114189/e86f7465-ff39-46e8-ad47-d15d5f3d48cf)

![image](https://github.com/go-gitea/gitea/assets/2114189/64596001-2cde-40ee-9b92-96d7d30dcdfd)

![image](https://github.com/go-gitea/gitea/assets/2114189/9103668a-f184-4e0b-a0e1-672daa00ed10)

![image](https://github.com/go-gitea/gitea/assets/2114189/356e4109-6784-47f9-a490-f64ab1831afb)

Reply:
Yes I know this style is used in other places. Still I prefer the indented style, but I guess you would reject me changing all these occurences :D